manifests/init.pp

```diff
@@ -28,15 +28,20 @@
 #
 class caddy (
 
-  String                         $install_path    = $caddy::params::install_path,
-  String                         $caddy_user      = $caddy::params::caddy_user,
-  String                         $caddy_group     = $caddy::params::caddy_group,
-  String                         $caddy_log_dir   = $caddy::params::caddy_log_dir,
-  String                         $caddy_tmp_dir   = $caddy::params::caddy_tmp_dir,
-  String                         $caddy_home      = $caddy::params::caddy_home,
-  Enum['personal', 'commercial'] $caddy_license   = 'personal',
-  Enum['on','off']               $caddy_telemetry = 'off',
-  String                         $caddy_features  = 'http.filter,http.git,http.ipfilter'
+  String                         $install_path     = $caddy::params::install_path,
+  String                         $caddy_user       = $caddy::params::caddy_user,
+  String                         $caddy_group      = $caddy::params::caddy_group,
+  String                         $caddy_log_dir    = $caddy::params::caddy_log_dir,
+  String                         $caddy_tmp_dir    = $caddy::params::caddy_tmp_dir,
+  String                         $caddy_ssl_dir    = $caddy::params::caddy_ssl_dir,
+  String                         $caddy_home       = $caddy::params::caddy_home,
+  Enum['personal', 'commercial'] $caddy_license    = 'personal',
+  String                         $caddy_account_id = '',
+  String                         $caddy_api_key    = '',
+  Enum['on','off']               $caddy_telemetry  = 'off',
+  String                         $caddy_features   = 'http.filter,http.git,http.ipfilter',
+  Integer                        $caddy_http_port  = $caddy::params::caddy_http_port,
```

Comment:
There exists  a port datatype that can be used: https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/types/port.pp

Reply:
stdlib is missing as a dependency in `metadata.json`, please add it:

```json
  "dependencies": [
    {
      "name": "puppetlabs/stdlib",
      "version_requirement": ">= 4.25.0 < 6.0.0"
    }
  ]
```

Reply:
added, will soon commit